qemu/tests/boot_with_different_vectors.py

```diff
@@ -58,17 +62,20 @@ def check_msi_support(session):
                 d_id = device.split()[0]
                 msi_check_cmd = params["msi_check_cmd"] % d_id
                 output = session.cmd_output(msi_check_cmd)
-                if vectors == 0 and output:
```

Comment:
I think the situation you mentioned has been covered by the following steps. Because the conditions of the test are quite special, the output conditions of ‘vectors==0‘, ‘vectors==1‘, and ‘vectors<=2’ are all different, so I think the current situation is reasonable. Please correct me if I am mistake, thanks.

Reply:
Ah, the premise of the following steps is that there is output, I am afraid if the output is empty, we will miss the checkpoint.

Reply:
Hi @PaulYuuu I added this check point, Please help me review it again. thanks a lot.